packages/tracing/src/browser/browsertracing.ts

```diff
@@ -195,8 +195,10 @@ export class BrowserTracing implements Integration {
 
   /** Create routing idle transaction. */
   private _createRouteTransaction(context: TransactionContext): Transaction | undefined {
+    const op = context.op ? `${context.op} ` : '';
+
     if (!this._getCurrentHub) {
-      logger.warn(`[Tracing] Did not create ${context.op} transaction because _getCurrentHub is invalid.`);
+      logger.warn(`[Tracing] Did not create ${op}transaction because _getCurrentHub is invalid.`);
```

Comment:
This should be inlined so it'll be stripped in the future.

```suggestion
      logger.warn(`[Tracing] Did not create ${context.op ? `${context.op} ` : ''}transaction because _getCurrentHub is invalid.`);
```